tests/Feature/PetControllerTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Contracts\PetService;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+
+use Mockery\MockInterface;
+use Tests\TestCase;
+
+use App\Models\Pet;
+
+class PetControllerTest extends TestCase
+{
+    private MockInterface $petServiceSpy;
+    private $pets = [];
+
+    private static function getPets() {
+        return [
+            Pet::make([
+                'id' => 1,
+                'name' => 'Fido',
+                'age' => 5,
+                'type' => Pet::PET_TYPE_DOG,
+            ]),
+            Pet::make([
+                'id' => 2,
+                'name' => 'Milo',
+                'age' => 3,
+                'type' => Pet::PET_TYPE_CAT,
+            ]),
+        ];
+    }
+
+    public function setUp(): void {
+        parent::setUp();
+
+        $this->pets = self::getPets();
+
+        $this->PetServiceSpy = $this->spy(PetService::class);
+    }
+
+    public function test_get_pets_returns_list() {
+        // arrange
+        $this->PetServiceSpy->shouldReceive('getPets')
+            ->once()
+            ->andReturn([
+                Pet::make([
```

Comment:
why not call the `getPets` method you declared above instead of rewriting this every time?